docs/content/v0.20/architecture.md

```diff
@@ -0,0 +1,36 @@
+# KubeStellar Architecture
+
+**NOTE**: This is only a very rudimentary start.
+
+## Higher Level
+
+At the higher level there are three modules.
+
+1. KubeFlex
```

Comment:
In a KubeStellar project I would assume KubeStellar should be the main and first module and the others should be some components it is using.

Reply:
Re-ordered in latest edition.